docs/hydra/guides/deploy-hydra-example.mdx

```diff
@@ -0,0 +1,494 @@
+use the following script to install Node.js 16.x on a Ubuntu system.
+
+```bash
+curl -sL https://deb.nodesource.com/setup_16.x -o /tmp/nodesource_setup.sh
+bash /tmp/nodesource_setup.sh
+apt-get install nodejs npm jq unzip
+# Install Node.js 16.x and other dependencies
+```
+
+Confirm the correct version of NodeJs is installed:
+
+```bash
+node -v
+v16.14.2
+```
+
+## Install PostgreSQL
+
+Install PostgreSQL by running the following command:
+
+```bash
+sudo apt install postgresql postgresql-contrib
+sudo -i -u postgres
+```
+
+Create the database:
+
+```bash
+createdb hydra
+```
+
+Change the default password encryption to a stronger one as
+[recommended by PostgreSQL](https://www.postgresql.org/docs/current/auth-password.html):
+
+```bash
+psql
+# Postgres command line
+ALTER SYSTEM SET password_encryption = 'scram-sha-256';
+# Change the default password encryption to stronger one
+SELECT pg_reload_conf();
+# Reload configuration
+```
+
+Let's create a user for Hydra (Use your own password / hash!):
+
+```bash
+CREATE USER hydra PASSWORD 'CHANGE-ME-INSECURE-PASSWORD';
+```
+
+We need to change the `/etc/postgresql/12/main/pg_hba.conf` file and append the following to enable `scram-sha-256` encryption:
+
+```bash
+host    all             all             127.0.0.1/32            scram-sha-256
+```
+
+Check your credentials against PostgreSQL:
+
+```bash
+psql -U hydra -W -h 127.0.0.1
+
+```
+
+Type or paste the password for the Ory Hydra user we created before. If everything works correctly you should see a prompt similar
+to this:
+
+```
+Password:
+psql (12.9 (Ubuntu 12.9-0ubuntu0.20.04.1))
+SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+Type "help" for help.
+
+hydra=>
+```
+
+Congratulations, you have installed and configured the PostgreSQL database. Next we will set up Ory Hydra.
+
+## Install Ory Hydra
+
+For security reasons, we need to create a new user with prohibited login:
+
+```bash
+useradd -s /bin/false -m -d /opt/hydra hydra
+```
+
+Create folders to hold the installation data and configuration files:
+
+```bash
+mkdir /opt/hydra/{bin,config}
+```
+
+Install Ory Hydra:
+
+<CodeBlock className="language-shell">{`
+cd /opt/hydra/bin
+# Download a new version of Ory Hydra
+wget https://github.com/ory/hydra/releases/download/${useLatestRelease('hydra')}/hydra_${useLatestRelease(
+  'hydra'
+)}-linux_64bit.tar.gz
+# Extract contents
+tar xfvz hydra_${useLatestRelease('hydra')}-linux_64bit.tar.gz
+# Remove redundant files
+rm *md
+rm LICENSE
+`}</CodeBlock>
+
+Download the [Quickstart](../5min-tutorial) configuration files:
+
+<CodeBlock className="language-shell">{`cd ../config
+wget https://raw.githubusercontent.com/ory/hydra/${useLatestRelease('hydra')}/contrib/quickstart/5-min/hydra.yml`}</CodeBlock>
+
+We need to add some configuration in `hydra.yml` to use our Postgres We need configure Ory Hydra to use the Postgres database.
```

Comment:
```suggestion
We need to add some configuration in `hydra.yml` to use our Postgres We need configure Ory Hydra to use the Postgres database.
```

Something went wrong here, not sure what this sentence was supposed to sound like?